src/js/disability-benefits/reducers/claims-list.js

```diff
@@ -12,10 +12,17 @@ const initialState = {
   consolidatedModal: false
 };
 
+// We want to sort claims without dates below claims with dates
+function phaseChangeDate(claim) {
+  return claim.attributes && claim.attributes.phaseChangeDate
+    ? claim.attributes.phaseChangeDate
+    : '0';
+}
+
 export default function claimsReducer(state = initialState, action) {
   switch (action.type) {
     case SET_CLAIMS: {
-      const sortedList = _.orderBy('attributes.updatedAt', 'desc', action.claims);
+      const sortedList = _.orderBy([phaseChangeDate, 'id'], 'desc', action.claims);
```

Comment:
Should we use the `evss_id` here? The vets.gov id will probably change alot as we have to sweep through and remove records every day. Might want to revisit which ID we use throughout the app so OK if we don't solve in this PR